src/Ztm.Zcoin.NBitcoin/Exodus/TransactionRetrievers/IExodusTransactionRetriever.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Ztm.Zcoin.NBitcoin.Exodus.TransactionRetrievers
+{
+    public interface IExodusTransactionRetriever
+    {
+        Type SupportType { get; }
```

Comment:
We should not use `Type` due to it will not support inheritance (e.g. `SimpleSendV2` derived from `SimpleSendV1`).

Reply:
Use `Id` instead then.